include/pfs/parsers.hpp

```diff
@@ -63,12 +63,16 @@ void parse_lines(
     }
 }
 
+std::string parse_key_value_line(const std::string& line);
+
 std::pair<std::string, bool> parse_filesystems_line(const std::string& line);
 std::pair<std::string, size_t> parse_meminfo_line(const std::string& line);
 
 zone parse_buddyinfo_line(const std::string& line);
 load_average parse_loadavg_line(const std::string& line);
 uptime parse_uptime_line(const std::string& line);
+sys_stat::cpu_info parse_stat_cpu_line(const std::string& line);
```

Comment:
Since we don't expect anyone to call this parser directly, we always declare per-line parsers as static methods inside the relevant cpp file.